src/components/Events.tsx

```diff
@@ -20,14 +22,25 @@ export const Events: React.FC = () => {
     fetchEvents();
   }, []);
 
+  const handleRedirect = (event: Event) => {
+    router.push({
+      pathname: `/event/${event.id}`,
+    });
+  };
+
   if (isLoading) {
     return <Loader />;
   }
 
   return (
     <ul className={style.list}>
       {events.map((event: Event) => (
-        <li key={event.id} className={style.event}>
+        <li
+          key={event.id}
+          className={style.event}
+          onClick={() => handleRedirect(event)}
+          style={{ cursor: "pointer" }}
```

Comment:
Nope. Don't use style. It is impossible to override; it creates multiple sources of truth for styling. You have that `style.event` over yonder where it belongs.

Reply:
resolved